packages/reporter/src/commands/command.tsx

```diff
@@ -471,6 +481,19 @@ class Command extends Component<Props> {
       }, 50)
     }
   }
+
+  _removeStudioCommand = (e: React.MouseEvent<HTMLElement, globalThis.MouseEvent>) => {
+    e.preventDefault()
+    e.stopPropagation()
+
+    const { model, events } = this.props
+
+    if (!model.isStudio) {
+      return
+    }
```

Comment:
Don't think this is necessary since it only is applied to the studio-command-remove icon when model.isStudio.